typescript/sdk/src/utils/zksync.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  ZkSyncArtifact,
+  loadAllZkArtifacts,
+} from '@hyperlane-xyz/core/zksync-artifacts';
```

Comment:
Please don't use import paths into the package like this. If what you need isn't exported from the package's index.ts, modify the package to have it exported